pkgs/tools/misc/seaborn-data/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, newScope, fetchFromGitHub, unzip, stdenvNoCC }:
+let
+  base = {
+    version = "master";
+    dontBuild = true;
+    meta = with lib; {
+      description = "Data repository for seaborn examples.";
+      homepage = "https://github.com/mwaskom/seaborn-data";
+      platforms = platforms.all;
+      maintainers = with maintainers; [ mbalatsko ];
+    };
+  };
+  makeNltkDataPackage = {pname, hash}:
```

Comment:
maybe change the name to makeSeabornDataPackage ?

also I just to make sure, this dataset is actually going to be used for another package you want to package later, right ?
And there will be a need to package several datasets in this repo, correct ?
briefly looking at the repo, it said it just contained examples.

Reply:
if you will only ever need exercise, and that you don't think it will be used for any other package, it might be best to just make a single stdenNoCC derivation inside the scikit-postdoc package.

Reply:
Oh it is really late, I see that I have to go sleeping. I'll rename it, thanks!

Reply:
I think, that a lot of packages could benefit now or in the future from it. It will not be used by only one package, it is official `seaborn` repo for datasets, from which `seaborn.download` downloads data

Reply:
fixed